.changeset/afraid-laws-speak.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro": minor
+---
+
+set build.assets directory during vite config setup
```

Comment:
Do you see some potential behaviour changes that this is marked a minor? If everything else still works as expected, this seems like a patch / bug fix to me.

Reply:
Oh right, per https://github.com/withastro/astro/pull/10705#pullrequestreview-1986028561 so the docs mention it it's a bug fix. Fixing.